### ISSUE TITLE
Windows High Contrast mode issues for 526 Confirmation page

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#95549]
## Issue Description
This issue was identified in the [accessibility audit of the review & confirmation pages ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94614)of the 526 form.

[**Expected severity**](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric): a11y-defect-1
**Page**: confirmation
**Environment**: Windows High Contrast Mode
**Estimated Level of Effort to fix**: Medium
**Design system component involved**: Not sure
**Relevant WCAG success criteria**: Best practice

### To Reproduce:
- https://devtoolstips.org/tips/en/emulate-forced-colors/

Issues in WHCM for the following items: 
 
- triangle alert icon for "It's taking us longer than expected to submit your claim" is not visible
![Image](https://github.com/user-attachments/assets/73d0fbcf-47eb-44de-bb46-8e9b240644aa)

- border box around Disability Compensation Claim form summary is not visible
- Print for your records button is missing border, possibly some other buttons missing borders as well.
![Image](https://github.com/user-attachments/assets/871d184b-9257-442b-b955-0223e3826955)

## Tasks
- [ ] Make items WHCM friendly by adding transparent borders or by using `currentColor` for SVGs
- [ ] Please update the "Fix Status" column in the [526 Accessibility Audit issues tracking doc](https://docs.google.com/spreadsheets/d/1paNxD798NSMrjZeUe9WwVQ4xRdl439ypKyPDPW3CXIo/edit?gid=2034611899#gid=2034611899) when ticket is started and completed.

## Acceptance Criteria
- [ ] Images & borders should be visible in Windows High Contrast Mode
- [ ] Accessibility Audit issue tracking doc is updated with ticket status
